app/assets/javascripts/admin/addon/templates/backups-index.hbs

```diff
@@ -1,26 +1,14 @@
 <div class="backup-options">
   {{#if localBackupStorage}}
-    {{uppy-backup-uploader done=(route-action "uploadSuccess") localBackupStorage=localBackupStorage}}
+    <UppyBackupUploader @done={{route-action "uploadSuccess"}} @localBackupStorage={{localBackupStorage}} />
   {{else}}
-    {{uppy-backup-uploader done=(route-action "remoteUploadSuccess")}}
+    <UppyBackupUploader @done={{route-action "remoteUploadSuccess"}} />
   {{/if}}
 
   {{#if site.isReadOnly}}
-    {{d-button
-      class="btn-default"
-      icon="far-eye"
-      action=(action "toggleReadOnlyMode")
-      disabled=status.isOperationRunning
-      title="admin.backups.read_only.disable.title"
-      label="admin.backups.read_only.disable.label"}}
+    <DButton @class="btn-default" @icon="far-eye" @action={{action "toggleReadOnlyMode"}} @disabled={{status.isOperationRunning}} @title="admin.backups.read_only.disable.title" @label="admin.backups.read_only.disable.label" />
```

Comment:
Cosmetic issue, but is there a chance we can avoid having these super-long one-liners? Or even keep the same indentation as before?

Reply:
Yeah, we'll have to live with this for a bit. It's gonna be resolved by CI/prettier changes in a couple of weeks (hopefully :) )